docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -258,6 +263,32 @@
     - name: Wait for Deployment Manager to be ready
       shell: KUBECONFIG=/etc/kubernetes/admin.conf /bin/kubectl wait --namespace=platform-deployment-manager --for=condition=Ready pods --selector control-plane=controller-manager --timeout=60s
       register: wait_for_deployment_manager
+      ignore_errors: true
+
+    - block:
+      - name: Search for the Deployment Manager pod name in case of failure
+        shell: |
+          kubectl -n platform-deployment-manager get pods | awk 'NR == 2 { print $1 }'
+        environment:
+          KUBECONFIG: "/etc/kubernetes/admin.conf"
+        register: deployment_manager_pod_name_after_fail
+
+      - name: Describe DM pod in case of download failed
+        shell: |
+          kubectl -n platform-deployment-manager describe pod "{{ deployment_manager_pod_name_after_fail.stdout }}" |
+          awk '/Failed/ || /error/  {print $0}'
+        environment:
+          KUBECONFIG: "/etc/kubernetes/admin.conf"
+        register: describe_dm_failed_pod
+
+      - name: Fail if deployment manager pod not ready
+        fail:
+          msg:
+            - "Deployment manager pods not ready"
+            - "Check helm overrides files and ensure you can pull from the specified sides."
```

Comment:
Check helm overrides files and ensure you can pull the image from the specified URL."